.github/dependabot.yml

```diff
@@ -53,3 +53,108 @@ updates:
     directory: /cmd/operator-opamp-bridge
     schedule:
       interval: weekly
+
+  - package-ecosystem: docker
+    directory: /autoinstrumentation/apache-httpd
+    schedule:
+      interval: daily
+
+  - package-ecosystem: docker
+    directory: /autoinstrumentation/dotnet
+    schedule:
+      interval: daily
+
+  - package-ecosystem: docker
+    directory: /autoinstrumentation/java
+    schedule:
+      interval: daily
+
+  - package-ecosystem: docker
+    directory: /autoinstrumentation/nodejs
+    schedule:
+      interval: daily
+
+  - package-ecosystem: npm
```

Comment:
Can we do the instrumentation package updates separately? I think there's a PR for changing this already, and it requires more careful consideration than everything else in this PR.